apps/trading-e2e/src/integration/deposits.feature

```diff
@@ -0,0 +1,110 @@
+Feature: Deposits to vega wallet
+
+  Background:
+    Given I navigate to deposits page
+
+  @todo
+  Scenario: Eth wallet connect message displayed if no ethereum wallet connected
+    Then I can see a message stating no wallet is connected
+    And the connect button is displayed
+
+  @todo
+  Scenario: Cannot deposit if approved amount is 0 (approval amount is 0)
+    And I connect my ethereum wallet
+    And I connect my vega wallet
+    When I set "0" tokens to be approved
+    And I approve the asset tokens
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
+      | Amount        | 50       |
+    And I click to the deposit the funds
+    And I approve the ethereum transaction
+    Then I can see the deposit is unsuccessful
+
+  @todo
+  Scenario: Cannot deposit if approved amount is lower than deposit amount
```

Comment:
suggestion non-blocking: "User is prompted to approve more if the deposit amount is greater than the approved amount (and approve !=0)"

Reply:
@campbellssource is there another journey here where the user will be prompted to approve more if the deposit is higher than approval amount ?

i thought the approval step is done prior to any deposit attempted

Reply:
it is this is if the first time approved I set it only to a bit more than my first deposit. but this scenario is a second attempt to deposit where the amount will over use my initial approved amount